src/commands/session.ts

```
@@ -159,3 +198,258 @@ export const sessionCommand = define({
+
+		// Extract session info from file path
+		const relativePath = path.relative(claudeDir, file);
+		const parts = relativePath.split(path.sep);
+		const sessionId = parts[parts.length - 2] ?? 'unknown';
+		const joinedPath = parts.slice(0, -2).join(path.sep);
+		const projectPath = joinedPath.length > 0 ? joinedPath : 'Unknown Project';
+		const sessionKey = `${projectPath}/${sessionId}`;
+
+		for (const line of lines) {
+			try {
+				const parsed = JSON.parse(line) as unknown;
```

ESLint

[error] Unsafe argument of type `any` assigned to a parameter of type `string`.

(ts/no-unsafe-argument)